### PR TITLE
Dev/kduran/vbs22 99 calendario actividades

### PR DESCRIPTION
**Pull request by kendallduranv**
Calendar front

### CODE REVIEW

File: src/app/components/trip-information/trip-information.component.ts [hunk 1]
```diff
@@ -133,16 +136,17 @@ export class TripInformationComponent {
...
 
-    this.tripNgModel.departure_date                 = this.initialForm.outbound_date;
-    this.tripNgModel.destination_city               = this.initialForm.q;
-    this.tripNgModel.return_date                    = this.initialForm.return_date;
+    this.tripNgModel.departureDate                  = this.initialForm.outbound_date;
+    this.tripNgModel.destination_city               = this.initialForm.q;  
+    this.tripNgModel.returnDate                     = this.initialForm.return_date;
     this.tripNgModel.budget                         = this.tripBudget.total;
-    // this.tripNgModel.currency                       = this.userCurrency[0]; //Falta implementar el servicio de currency
+    this.tripNgModel.currency                       = Number(this.userInfo.currencyId ?? 0);; //Falta implementar el servicio de currency
```
Comment (nefi-cenfo) [inline src/app/components/trip-information/trip-information.component.ts:143]: You can move out the nullish default operator. It seems there is a repeated semicolon at the end.

File: src/app/interfaces/flights.interface.ts [hunk 2]
```diff
@@ -33,17 +33,22 @@ export interface IVoiageFlight{
...
+    layovers                        : IFlights[];
 
     //Propiedades de auditoria
-    creation_datetime               : Date;
-    creation_responsible            : number;
-    lastUpdate_datetime?            : Date;
-    update_responsible?             : number;
-    operational                     : number;
+    created_at                      : Date;
+    // creation_responsible            : number;
```
Comment (nefi-cenfo) [inline src/app/interfaces/flights.interface.ts:50]: Be aware of the lines commented.

File: src/app/interfaces/user.interface.ts [hunk 3]
```diff
@@ -19,4 +19,8 @@ export interface IUser {
   roleId?: number;
   otp?: string;
 }
-  
\ No newline at end of file
+  
+
+export interface IUserId {
```
Comment (nefi-cenfo) [inline src/app/interfaces/user.interface.ts:24]: What is the purpose of this interface?

File: src/app/services/calendar-event.service.ts [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+})
+export class CalendarEventService extends BaseService<ICalendarEvent> {
+
+    protected override source: string = 'calendarEvent';
+
+    private calendarEventSignal = signal<ICalendarEvent[]>([]);
+
+
+    get calendarEvent$() {
+        return this.calendarEventSignal;
```
Comment (nefi-cenfo) [inline src/app/services/calendar-event.service.ts:19]: Please consider to return the signal as readonly.

File: src/app/services/calendar-event.service.ts [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    };
+
+
+    getAllSignal(userId:IUserId) {
+        
+        this.bringInfoWithParams(userId).subscribe({
+            next: (response: any) => {
+                
+                this.calendarEventSignal.set(response);
+                console.log(response);
```
Comment (nefi-cenfo) [inline src/app/services/calendar-event.service.ts:46]: Please remove this console log.

**Review by nefi-cenfo — DISMISSED** [review]

**Review by ichivancrCenfotec — APPROVED** [review]

**Review by nefi-cenfo — APPROVED** [review]

**Pull request merged**